autoload/preview_markdown.vim

```diff
@@ -100,7 +100,8 @@ function! preview_markdown#preview() abort
     endif
 
     let s:preview_buf_nr = term_start(cmd, opt)
-
+	
+	sleep 1
```

Comment:
If need to sleep, use job exit callback is more better.

e.g

Add opt

```
'exit_cb': function('s:remove_tmp', [tmp]),
```

Add `s:remove_tmp` function.

```
function! s:remove_tmp(tmp, channel, msg) abort
  call delete(tmp)
endfunc

Reply:
Okay!!